pulseapi/entries/serializers/curriculum.py

```diff
@@ -0,0 +1,26 @@
+from ..models import Entry
+from .news import EntrySerializerWithCreators
+
+
+class CurriculumEntrySerializer(EntrySerializerWithCreators):
+    """
+    Serializes an entry based on the fact that we
+    know that it is a curriculum item.
+    
+    This has the same requirements as news items.
+    """
+
+    class Meta:
+        model = Entry
+        read_only_fields = fields = (
+            'id',
+            'title',
+            'content_url',
+            'thumbnail',
+            'is_bookmarked',
+        )
+        extra_kwargs = {
+            'url': {
```

Comment:
not sure I understand what this is for? `url` doesn't seem to be a field in the `Entry` model

Reply:
Gah, that should have been `content_url`

Reply:
is there a reason it's in `extra_kwargs`?